libinteract/acPSN.py

```diff
@@ -237,11 +264,17 @@ def _get_i_const(self,
     def _get_residues(self,
```

Comment:
what happens here if we have e.g. solvent or other molecules in the xtc, should we expect it to return those residues as well?

Reply:
if we're running in non-permissive mode, then the program willl just exit. Then either the user will run in permissive mode or add the normalisation factors manually

Reply:
It shouldn't be a problem since only protein residues get selected later for the acPSN calculation. For when you run in non-permissive mode and encounter residues where no normalisation factor is provided, I think exiting should be the default behaviour.